Mathlib/Data/Matroid/Rank/Finite.lean

```diff
@@ -103,6 +106,7 @@ lemma Indep.subset_finite_isBasis_of_subset_of_isRkFinite (hI : M.Indep I) (hIX
     (hX : M.IsRkFinite X) (hXE : X ⊆ M.E := by aesop_mat) : ∃ J, M.IsBasis J X ∧ I ⊆ J ∧ J.Finite :=
   (hI.subset_isBasis_of_subset hIX).imp fun _ hJ => ⟨hJ.1, hJ.2, hJ.1.finite_of_isRkFinite hX⟩
 
+@[simp]
```

Comment:
Optional, but you could instead mark `isRkFinite_of_finite` as simp instead, which would imply this one. (This might be a small performance hit, but I think it's specific enough to not be a problem)

Reply:
Good idea